crossai/loader/_audio.py

```diff
@@ -35,7 +35,7 @@ def wavfile_reader(filename):
     signal = (signal - np.min(signal)) / (np.max(signal) - np.min(signal))
     signal = signal * (n_range[1] - n_range[0]) + n_range[0]
 
-    return signal
+    return signal, filename 
```

Comment:
In the `wavfile_reader` function, `filename` variable currently returns the full filename (path and extension). For clarity and data management issues, I recommend modifying the function to return only the base filename without the extension. This can be achieved by using `os.path.splitext` and `os.path.basename`:

`return signal, os.path.splitext(os.path.basename(filename))[0]`